### PR TITLE
new pr

### PR DESCRIPTION
**Pull request by RandolphTang**
Author: <name>

## What changes were made?

All officer info is updated

### CODE REVIEW

**Review by elimelt — CHANGES_REQUESTED** [review]

File: package.json [hunk 1]
```diff
@@ -18,6 +19,7 @@
     "react-scripts": "5.0.1",
     "react-social-media-embed": "^2.3.5",
     "react-transition-group": "^4.4.5",
+    "resend": "^4.0.0",
```
Comment (ShawnCollinge) [inline package.json:22]: is this being used anywhere?

File: src/Components/App.js [hunk 2]
```diff
@@ -1,35 +1,52 @@
...
+            <ExternalRedirect to="https://www.instagram.com/swecc.uw/" />
+          }
+        />
+        <Route
+          path="/mailing-list"
+          element={
+            <ExternalRedirect to="https://mailman11.u.washington.edu/mailman/listinfo/sweccmailinglist" />
+          }
+        />
+        <Route path="/join-Now" element={<JoinNow />} />
```
Comment (ShawnCollinge) [inline src/Components/App.js:46]: should probably lowercase the `N` to follow the rest.

File: src/Components/Events.js [hunk 3]
```diff
@@ -17,38 +21,60 @@ function Events() {
...
+                : ""
+              : "RSVP releasing soon✨"}
+          </p>
         )}
       </div>
     </div>
   ));
 
-  const postId = ["C2-o7roPbfN", "C23PyAWPYdW", "C2boAR4LeRy", "CzVNKp6r9hn", "CzVLbJWLwsI", "Cy4WTIJvVvB", "CyzVzMTJlhQ", "CyRondZPVVS", "CyCms3mLOLb", "Cxum20yOlkQ", "CxrxWcQLpSb"];
+  const postId = [
```
Comment (ShawnCollinge) [inline src/Components/Events.js:36]: Is there a reason for hardcoding postIds? Are we not just wanting the most recent `n` posts? Maybe you can't answer this but maybe @elimelt can?

File: src/Components/Header.js [hunk 4]
```diff
@@ -1,28 +1,34 @@
...
       <div className="nav-bar">
         <ul>
-          <NavLink to="/" className="nav-link">HOME</NavLink>
-          <NavLink to="/Events" className="nav-link">EVENTS</NavLink>
-          <NavLink to="/About" className="nav-link">ABOUT</NavLink>
-          <NavLink to="/Join-Now" className="join-now-btn nav-link">JOIN NOW</NavLink>
+          <NavLink to="/" className="nav-link">
+            HOME
+          </NavLink>
+          <NavLink to="/Events" className="nav-link">
```
Comment (ShawnCollinge) [inline src/Components/Header.js:19]: Does this link work? You lowercased it in the routes.

File: src/Components/Header.js [hunk 5]
```diff
@@ -1,28 +1,34 @@
...
+            EVENTS
+          </NavLink>
+          <NavLink to="/About" className="nav-link">
```
Comment (ShawnCollinge) [inline src/Components/Header.js:22]: ditto here

File: src/Components/Header.js [hunk 6]
```diff
@@ -1,28 +1,34 @@
...
+            ABOUT
+          </NavLink>
+          <NavLink to="/Join-Now" className="join-now-btn nav-link">
```
Comment (ShawnCollinge) [inline src/Components/Header.js:25]: ditto here.

File: src/Components/Home.js [hunk 7]
```diff
@@ -63,85 +69,112 @@ function HomePage() {
...
+          className="posts-carousel--right"
+          onClick={() => scroll("right")}
+        >
+          <FaChevronRight />
+        </button>
+      </div>
+      <div className="posts" ref={scrollContainerRef}>
+        <InstagramEmbed
+          className="instaPost slide-up"
+          url="https://www.instagram.com/p/C7uRAftvEKV/?utm_source=ig_web_copy_link"
```
[the remark this answers is not in the document]
Comment (ShawnCollinge) [reply src/Components/Home.js:112]: Why are we hardcoding these urls?

File: src/Components/Navbar.js [hunk 8]
```diff
@@ -102,21 +93,36 @@ function Navbar() {
...
+          {/*  icon={<FcViewDetails />}*/}
+          {/*  route={"/About"}*/}
+          {/*  closeExpand={closeExpand}*/}
+          {/*  tooltip="About"*/}
+          {/*  label="About"*/}
+          {/*/>*/}
           <NavItem
             icon={<FcConferenceCall />}
-            route={"/About"}
+            route={"/Officers"}
```
Comment (ShawnCollinge) [inline src/Components/Navbar.js:105]: This should also be lowercase.

File: src/Components/Navbar.js [hunk 9]
```diff
@@ -102,21 +93,36 @@ function Navbar() {
...
             expand={expand}
             closeExpand={closeExpand}
             tooltip="Officers"
             label="Officers"
           />
+          <NavItem
+            icon={<FaHeart />}
+            route={"/Sponsor"}
```
Comment (ShawnCollinge) [inline src/Components/Navbar.js:113]: lowercase?

File: src/Components/Navbar.js [hunk 10]
```diff
@@ -221,7 +195,7 @@ function DropdownMenu(props) {
           <NavLink to="/" onClick={props.closeExpand}>
             <DropdownItem leftIcon={<FcHome />}>Home</DropdownItem>
           </NavLink>
-          <NavLink to="/About" onClick={props.closeExpand}>
+          <NavLink to="/Officers" onClick={props.closeExpand}>
```
Comment (ShawnCollinge) [inline src/Components/Navbar.js:198]: lowercase?

File: src/Components/Officers.js [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+    <div>
+      <div>
+        <div className="about-message-section">
+          <h1>Our Officers</h1>
+        </div>
+        <div className="Officer-content">
+          <nav className="officer-sidebar">
+            <h2>Year</h2>
+            <ul>
+              {[2024, 2023, 2022, 2021].map((year) => (
```
Comment (ShawnCollinge) [inline src/Components/Officers.js:37]: Maybe we can not hardcode years, can check if after september then add the next year? Starting with 2021.

**Review by ShawnCollinge — COMMENTED** [review]
Why do we have header.js and nav-bar.js with the same links? I didn't dive too deep into it but something to maybe look into for refactoring.

File: react-email-starter/emails/notion-magic-link.tsx [hunk 12]
```diff
@@ -0,0 +1,150 @@
...
+  </Html>
+);
+
+NotionMagicLinkEmail.PreviewProps = {
+  loginCode: "sparo-ndigo-amurt-secan",
+} as NotionMagicLinkEmailProps;
+
+export default NotionMagicLinkEmail;
+
+const main = {
```
Comment (ShawnCollinge) [inline react-email-starter/emails/notion-magic-link.tsx:96]: Why don't we move this css into it's own spot and I think the css can be shared across this and plaid-verify?

File: src/Components/Home.js [hunk 13]
```diff
@@ -83,53 +82,14 @@ function HomePage() {
...
-          className="instaPost slide-up"
-          url="https://www.instagram.com/p/CxrxWcQLpSb"
-        />
-      </div>
-    </div>
-  );
+  function intPostsWidget() {
+    return (
+      <BeholdWidget
+        onLoad={() => console.log("Loaded!")}
```
Comment (ShawnCollinge) [inline src/Components/Home.js:88]: Shouldn't have console.logs in prod please.

**Pull request closed without merging**